cloud/resource_cloud_environment.go

```diff
@@ -161,19 +175,43 @@ func resourceCloudEnvironmentCreate(ctx context.Context, d *schema.ResourceData,
 	if err != nil {
 		return diag.FromErr(fmt.Errorf("ERROR_CREATE_CLOUD_ENVIRONMENT: %w", err))
 	}
-	if ce.Status.Conditions != nil {
-		ready := false
+
+	if ce.Status.Conditions == nil {
```

Comment:
Do we still need these block since we've added retry at https://github.com/streamnative/terraform-provider-streamnative/pull/41/files#diff-c02472e6df317b435706b287c76b320eea0b27f53a2bc8c447058fcc2be79cf6R196

Reply:
Yes, when the resource is created it doesn't immediately have a status set so we need to poll to get it regardless of whether we check that it's true or false